PerfectNumberTest.java

```diff
@@ -16,5 +17,11 @@ public void should_return_true_As_1_is_factorOf_6() {
         PerfectNumber perfectNumber = new PerfectNumber();
         assertTrue(perfectNumber.isFactor(1,6));
     }
+    
+    @Test
+    public void should_return_false_As_4_is_not_factorOf_6() {
+        PerfectNumber perfectNumber = new PerfectNumber();
+        assertFalse(perfectNumber.isFactor(4,6));
+    }
 

```

Comment:
This is implementation based approach. You don't need to bother about factors at this moment. We have not yet established a single test to find a perfect number yet.